testcases/PowerNVDump.py

```diff
@@ -135,7 +138,28 @@ def setup_test(self, dump_place="local"):
                 "ls -l /var/crash | grep '^d'| awk '{print $9}'")
         if dump_place == "net":
             self.crash_content = self.c.run_command(
-                "ssh root@%s \"ls -l %s | grep '^d'\" | awk '{print $9}'" % (self.dump_server_ip, self.dump_path))
+                "ssh -i %s root@%s \"ls -l %s | grep '^d'\" | awk '{print $9}'" % (self.rsa_path, self.dump_server_ip, self.dump_path))
+            log.debug("crash content is %s"%(self.crash_content))
+ 
+    def setup_pwdless_auth(self):
+        # Setup Pulickey authentication for Kdump over ssh and NFS
+        try:
+            self.c.run_command("ls %s"% self.rsa_path)
+            self.c.run_command("rm -f %s"% self.rsa_path)
```

Comment:
No need of ls command just `rm -f file` is enough,  meaning it removes file if exists and it will not fail command if file not exists